sigle/src/ui/Button.ts

```diff
@@ -13,12 +13,17 @@ export const Button = styled('button', {
     pointerEvents: 'none',
   },
 
+  '@xl': {
+    fontSize: '$2',
+  },
+
   variants: {
     size: {
       md: {
         px: '$3',
         py: '$1',
         br: '$1',
+        maxHeight: 28,
```

Comment:
if we set this max height prop here we can't have multi-line buttons anymore, so doesn't seems like the right way to fix the problem no ?

Reply:
hmm yeah that's true - will see with Quentin if it's better to update the design then as we inspected on devtools and all properties are the same so we have no idea where the extra pixels are coming from 😅

Reply:
that's a really strange one, not the first time I see smth like this 😓